lib/toTestResult.ts

```diff
@@ -0,0 +1,83 @@
+import type { TestResult } from '@jest/test-result';
+import type { Milliseconds } from './types';
+
+interface Options {
+  stats: {
+    failures: number;
+    passes: number;
+    pending: number;
+    todo: number;
+    start: number;
+    end: number;
+  };
+  skipped: boolean;
+  errorMessage?: string | null;
+  tests: Array<{
+    duration?: Milliseconds | null;
+    errorMessage?: string;
+    testPath?: string;
+    title?: string;
+  }>;
+  jestTestPath: string;
+}
+
+function getPerfStats({ stats }: Options): TestResult['perfStats'] {
+  const start = new Date(stats.start).getTime();
+  const end = new Date(stats.end).getTime();
+  const runtime = end - start;
+  // TODO: determine `slow` by using `runtime / 1000 > config.slowTestThreshold`
```

Comment:
yeah, we should. Can do as a follow-up, tho 👍

Reply:
#47